streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -452,7 +452,9 @@ public class StreamsConfig extends AbstractConfig {
     /** {@code commit.interval.ms} */
     @SuppressWarnings("WeakerAccess")
     public static final String COMMIT_INTERVAL_MS_CONFIG = "commit.interval.ms";
-    private static final String COMMIT_INTERVAL_MS_DOC = "The frequency in milliseconds with which to save the position of the processor." +
+    private static final String COMMIT_INTERVAL_MS_DOC = "The frequency in milliseconds with which to commit processing progress." +
+        " For at-least-once processing, committing means to save the position (ie, offsets) of the processor." +
+        " For exactly-once processing, it means to commit the transaction which includes to save the position." +
```

Comment:
Should we also mention that the interval with which transactions are committed also affects the latency downstream?

Reply:
Not sure -- would we go down the rabbit hole?

Reply:
When I read the updated doc, the first question that comes to my mind is, what "commit the transaction" actually means. I would not like to leave a user with that question in mind. You could add something like 

```suggestion
        " For exactly-once processing, it means to commit the transaction which includes to save the position and to make the committed data in the output topic visible to consumers with isolation level read_committed." +
```

There are more to transactions that just that, but I think those are the main points.

Mine is just a suggestion, if you think it is not needed, I am fine.

Reply:
+1 to what Bruno said -- the current wording, `"to save the position"` is more like an implementation detail (sort of) but we should be explaining this in terms of what it actually affects for the user/what's visible to them, which is as Bruno describes to make the data actually readable